ui/page/staking/stake_overview.go

```diff
@@ -208,15 +208,14 @@ func (pg *Page) layoutDesktop(gtx C) D {
 
 	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 		layout.Rigid(pg.stakePriceSection),
+		layout.Rigid(pg.stakeStatisticsSection),
 		layout.Flexed(1, func(gtx C) D {
-			return layout.Inset{Top: values.MarginPadding8}.Layout(gtx, func(gtx C) D {
-				if pg.showMaterialLoader {
-					gtx.Constraints.Min.X = gtx.Constraints.Max.X
-					return layout.Center.Layout(gtx, pg.materialLoader.Layout)
-				}
-				return pg.scroll.List().Layout(gtx, 1, func(gtx C, i int) D {
-					return pg.ticketListLayout(gtx)
-				})
+			if pg.showMaterialLoader {
+				gtx.Constraints.Min.X = gtx.Constraints.Max.X
+				return layout.Center.Layout(gtx, pg.materialLoader.Layout)
+			}
+			return pg.scroll.List().Layout(gtx, 1, func(gtx C, i int) D {
+				return pg.ticketListLayout(gtx)
 			})
```

Comment:
it is not related to whether there is a ticket or not, the scroll bar is displayed because the display space is not enough, if you drag the space down you will see it no longer has a scroll bar